pages/doc/trace_data_sampling.md

```diff
@@ -43,54 +61,52 @@ A sampling strategy is a mechanism for selecting which traces to forward to Wave
-* If a trace has spans from multiple microservices, and you set up different sampling rates for those microservices. 
-
+* If a trace has spans from multiple services, and you set up different sampling rates for those services. 
 
-### When Sampling Strategies are Combined
+### Result of Combining Explicit Sampling Strategies
 
-You can combine rate-based sampling and duration-based sampling in the same microservice. Doing so causes Wavefront to ingest the union of the spans that are selected by each sampler.
+You can combine rate-based sampling and duration-based sampling in the same service. Doing so causes Wavefront to ingest the union of the spans that are selected by each sampler.
 
-For example, suppose you set the sampling rate to 20% and the sampling duration to 45ms for the same microservice. This causes Wavefront to receive:
-* 20% of the traces generated by that microservice, regardless of the length of their spans.
+For example, suppose you set the sampling rate to 20% and the sampling duration to 45ms for the same service. This causes Wavefront to receive:
+* 20% of the traces generated by that service, regardless of the length of their spans.
 * Any additional spans outside of that 20% that are longer than 45ms. 
 
 As a result, the ingested sample will contain somewhat more than 20% of the generated traces, with some spans that are shorter than 45ms.
 
 **Note:** A span that contains an error is always sent to Wavefront, the regardless of the span's duration or whether it falls in a specified sampling percentage. 
 
+## Setting Up Explicit Sampling Through the Proxy
 
-## Ways to Set Up Sampling
-You can set up a sampling strategy using either of the following methods:
-
-* [Configure sampling on a Wavefront proxy](#setting-up-sampling-through-the-proxy).  
-* [Configure sampling in your instrumented application code](#setting-up-sampling-in-your-code).  
-
-Choose the [Wavefront proxy](proxies_installing.html) for sampling when you want to:
-* Use a single sampling strategy to coordinate the sampling for all applications that use the same proxy. 
-* Configure sampling with minimal effort.
-* Improve the likelihood of ingesting complete traces. 
-
-Choose sampling in your instrumented code when you want to:
-* Reduce the performance impact of span reporting on your application. 
-* Use [direct ingestion](direct_ingestion.html) (so no proxy).
-* Configure sampling on a per-process basis, for example, when you expect spans from the services in different processes to have different characteristics.
-
-
-## Setting Up Sampling Through the Proxy
-
-You can set up sampling strategies through a [Wavefront proxy](proxies_installing.html) by adding the sampling properties to the proxy's configuration file.
+You can set up explicit sampling strategies through a [Wavefront proxy](proxies_installing.html) by adding the sampling properties to the proxy's configuration file.
```

Comment:
Maybe link to https://docs.wavefront.com/proxies.html instead of proxies_installing? What's the primary purpose of the link?

Reply:
+1.  Updated and added a note on the proxy version this feature is supported.